applications/api-cat/src/main/resources/apidocument.mapping.json

```diff
@@ -37,36 +37,22 @@
         }
       },
       "title": {
-        "properties": {
-          "no": {
-            "analyzer": "norwegian",
-            "type": "string",
-            "fields": {
-              "raw": {
-                "type": "string",
-                "index": "not_analyzed"
-              }
-            }
-          }
-        }
+        "analyzer": "norwegian",
+        "type": "string"
```

Comment:
Dette er feil. Man skal ikke lengre bruke type: string. Men heller text. Se https://www.elastic.co/blog/strings-are-dead-long-live-strings

Reply:
Men for å få sortert på tittel, må vi ha et ekstra felt: "properties": {
          "name": {
            "type": "text",
            "fielddata": true,
            "analyzer": "norwegian",
            "fields": {
              "raw": {
                "type": "keyword"
              }
            }
          },